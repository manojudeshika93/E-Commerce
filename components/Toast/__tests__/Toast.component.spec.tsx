import { render } from '@testing-library/react-native';
import * as React from 'react';

import { ToastHost } from '../Toast.component';

describe('ToastHost Component', () => {
  const getRenderedScreen = () => render(<ToastHost />);

  it(`should render ToastHost correctly`, () => {
    const renderer = getRenderedScreen();
    const renderTree = renderer.toJSON();
    expect(renderTree).toMatchSnapshot();
  });
});
