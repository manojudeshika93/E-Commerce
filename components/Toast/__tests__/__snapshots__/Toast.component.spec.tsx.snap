// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToastHost Component should render ToastHost correctly 1`] = `
<View
  collapsable={false}
  pointerEvents="box-none"
  style={
    {
      "left": 0,
      "opacity": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
      "transform": [
        {
          "translateY": -48,
        },
      ],
      "zIndex": 99,
    }
  }
/>
`;
